crates/chainspec/src/api.rs

```diff
@@ -130,3 +130,13 @@ impl EthChainSpec for ChainSpec {
         self.paris_block_and_final_difficulty.map(|(_, final_difficulty)| final_difficulty)
     }
 }
+
+/// Trait representing the current capacities of the fork.
+pub trait EthereumCapacities: EthereumHardforks {
```

Comment:
What's the rationale behind the `Capacities` naming? I'm not sure if it accurately reflects the context. Maybe something like `EthereumPayloadAttributesSpec` or `EthereumPayloadAttributesCapabilities`. What do you think?

Reply:
`EthereumCapabilities` makes sense to me. I would suggest to avoid using `eip_*` format as we then have to look up the eip to get the context.

Reply:
Feel free to merge once updated